### PR TITLE
Update documentation for running Unity builds on Windows VMs

### PR DESCRIPTION
**Pull request by nevercode-kevin**
- Change from Bash script to Python to activate the license, export apps, and return license so it works on Windows VMs too. 
- Add workflow example of how to build Unity Windows app.

### CODE REVIEW

File: content/yaml-quick-start/building-a-unity-app.md [hunk 1]
```diff
@@ -328,87 +342,79 @@ The best way to perform this step is to run an external shell script, which mean
...
-
-exit 0
+4. Add the script below to `export_unity.py` and check the file into the root of your repository.
+
+```python
+import os
+import platform
+import sys
+
+# Check if the app type argument has been provided e.g python3 export_unity.py windows | ios | android
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-unity-app.md:355]:
```suggestion
# Check if the app type argument has been provided, e.g python3 export_unity.py windows | ios | android
```

File: content/yaml-quick-start/building-a-unity-app.md [hunk 2]
```diff
@@ -328,87 +342,79 @@ The best way to perform this step is to run an external shell script, which mean
...
+if len(sys.argv) >= 2:
+    APP_TYPE = sys.argv[1].lower()
+    print(f'APP_TYPE set to: {APP_TYPE}')
+else:
+    print('APP_TYPE not set. End process.')
+    sys.exit()
+
+# Check environment variables have been set
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-unity-app.md:363]: Probably missing some word here, e.g. Check if/that..

File: content/yaml-quick-start/building-a-unity-app.md [hunk 3]
```diff
@@ -328,87 +342,79 @@ The best way to perform this step is to run an external shell script, which mean
...
+    UNITY_BIN = vars['UNITY_HOME'] + '/Contents/MacOS/Unity' 
+    LOG_DIR = '/Logs/'
+elif platform == 'Windows':
+    UNITY_BIN = vars['UNITY_HOME'] + '\\Unity.exe'
+    LOG_DIR = '\\Logs\\'
+elif platform == "Linux":
+    print("Build for Linux on Mac or Windows. End Process.")
+    sys.exit()
+
+# Check UNITY_BIN has been set
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-unity-app.md:384]: Check that...

File: content/yaml-quick-start/building-a-unity-app.md [hunk 4]
```diff
@@ -491,12 +497,25 @@ workflows:
...
-              ./export_unity.sh android        
+              python3 export_unity.py android        
       artifacts:
           - android/*.aab
       publishing:
           google_play:
             # See the following link for information regarding publishing to Google Play - https://docs.codemagic.io/publishing-yaml/distribution/#google-play
             credentials: $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS
-            track: alpha   # Any default or custom track that is not in ‘draft’ status    
+            track: alpha   # Any default or custom track that is not in ‘draft’ status
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-unity-app.md:507]: Just saying that it's now also possible to publish as draft

**Review by helinanever — COMMENTED** [review]
Added minor comments.

File: content/yaml-quick-start/building-a-unity-app.md (hunk 2 above)
Comment (nevercode-kevin) [reply content/yaml-quick-start/building-a-unity-app.md]: Added 'that'

File: content/yaml-quick-start/building-a-unity-app.md (hunk 3 above)
Comment (nevercode-kevin) [reply content/yaml-quick-start/building-a-unity-app.md]: Added 'that'

File: content/yaml-quick-start/building-a-unity-app.md (hunk 4 above)
Comment (nevercode-kevin) [reply content/yaml-quick-start/building-a-unity-app.md]: Removed comment about 'draft' status.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**